lib/install/web.rb

```diff
@@ -1,13 +1,19 @@
 def node_version
-  ENV["NODE_VERSION"] || `node --version`[/\d+\.\d+\.\d+/]
+  version = ENV["NODE_VERSION"] || `node --version`[/\d+\.\d+\.\d+/]
+
+  return if version.blank?
+
+  Gem::Version.new(version)
 end
 
 def node_not_installed?
-  !node_version.present?
+  node_version.blank?
```

Comment:
I changed this to avoid the double negative of `not_installed` and `!(...).present?`.